PSFramework/internal/loggingProviders/gelf.provider.ps1

```diff
@@ -0,0 +1,159 @@
+﻿# Action that is performed on registration of the provider using Register-PSFLoggingProvider
+$registrationEvent = {
+
+}
+
+#region Logging Execution
+# Action that is performed when starting the logging script (or the very first time if enabled after launching the logging script)
+$begin_event = {
+    $gelf_gelfserver = Get-PSFConfigValue -FullName 'PSFramework.Logging.GELF.GelfServer'
+    $gelf_port = Get-PSFConfigValue -FullName 'PSFramework.Logging.GELF.Port'
+    $gelf_encrypt = Get-PSFConfigValue -FullName 'PSFramework.Logging.GELF.Encrypt'
+
+    $gelf_paramSendPsgelfTcp = @{
+        'GelfServer' = $gelf_gelfserver
+        'Port' = $gelf_port
+        'Encrypt' = $gelf_encrypt
+    }
```

Comment:
This really should be in the `start` event, since otherwise changes to the configuration after logging has commenced would be ignored